test/source/mock/backend/backend-data.ts

```diff
@@ -97,7 +97,7 @@ export class BackendData {
         "flags": [
           "FORBID_STORING_PASS_PHRASE",
         ],
-        "in_memory_pass_phrase_session_length": 3
+        "in_memory_pass_phrase_session_length": 10
```

Comment:
Increased session length because when tests run serially in semaphoreci, passphrase might expire(3 seconds length) before getting passphrase.